Node/test/TeamsMessage.js

```diff
@@ -127,6 +127,66 @@ describe('TeamsMessage', function () {
       });
       connector.processMessage('start');
 		});
+
+		it('should mention user with UserMention', function () {
+			var connector = new builder.ConsoleConnector();
+      var bot = new builder.UniversalBot(connector);
```

Comment:
Please fix the indentation in this (**entire**) file.

Reply:
fixed